data/governance/proposal_fund_store.go

```diff
@@ -125,39 +132,112 @@ func (store *ProposalFundStore) IsFundedByFunder(id ProposalID, funder keys.Addr
 	return haveFunderAddress
 }
 
+func (pf *ProposalFundStore) GetCurrentFundsForProposal(proposalID ProposalID) (*balance.Amount, error) {
```

Comment:
I was thinking maybe we need to differentiate two scenarios:
1. there is no such record(this maybe means serious problem)
2. or there is such record, but we already totally withdrawn it, the record is left here with 0 amount just for future audit purpose.

Reply:
I mean not causing a serious problem. but revealing potential problem if there is any.
for example,
if there is no such record when it should exit, that means we accidentally deleted this record, which should not happen.
because by the design I made in deleteAllFunds, it will delete all individual records, and set total records to 0, not delete total records.

99.99% we won't hit that error, though I just want to show more info and reveal the potential problem in the process.
I'm also ok to remove this.

Reply:
in the blockchain world, zero value means "doesn't exist" or vise versa. so if it's zero where it shouldn't be, then you know it's a problem. if the code you write have the potential to create problem you don't want, you shouldn't write code in that way.

Reply:
ok I'll remove that error.